.editorconfig

```diff
@@ -0,0 +1,219 @@
+root = true
+
+# default settings:
+[*]
+insert_final_newline = true
+indent_style = space
+indent_size = 2
```

Comment:
Please don't specify an indent_size here; that has too many unintended consequences